src/__fixtures__/schema-kitchen-sink.graphql

```diff
@@ -56,6 +56,12 @@ extend interface Bar {
 
 extend interface Bar @onInterface
 
+interface Baz implements Bar {
```

Comment:
Since it's a kitchen sink it's better to implement couple of interfaces e.g. `implements Foo & Bar`.
Also please add an example of interface extension that adds a new interface

Reply:
Done and done.